src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/CaseServiceFixture.java

```diff
@@ -27,4 +27,5 @@
   public static final Date A_REQUEST_DATE_TIME = new Date();
   public static final String AN_AGENT_ID = "123";
   public static final String A_QUESTIONNAIRE_ID = "566786126";
+  public static final String A_UAC = "dummy-uac-value";
```

Comment:
You can ignore this if you like but I don't think TOWN and POSCODE are used anywhere

Reply:
Me neither, I didn't put them there, removed.